pages/contact-us.md

```diff
@@ -45,10 +45,10 @@ follow discussions as they happen.
 {: .table }
 | Name              | Description                                      | Read | Follow | Post |
 |-------------------|--------------------------------------------------|------|--------|------|
-| **user**          | General user questions, help, and announcements  | [<span class="glyphicon glyphicon-search"/> Archive][U_A] | [<span class="glyphicon glyphicon-plus"/> Subscribe][U_SU] [<span class="glyphicon glyphicon-remove"/> Unsubscribe][U_UN] | [<span class="glyphicon glyphicon-envelope"/> Post][U_P] |
-| **dev**           | Contributor discussions and development activity | [<span class="glyphicon glyphicon-search"/> Archive][D_A] | [<span class="glyphicon glyphicon-plus"/> Subscribe][D_SU] [<span class="glyphicon glyphicon-remove"/> Unsubscribe][D_UN] | [<span class="glyphicon glyphicon-envelope"/> Post][D_P] |
-| **commits**       | Code changes                                     | [<span class="glyphicon glyphicon-search"/> Archive][C_A] | [<span class="glyphicon glyphicon-plus"/> Subscribe][C_SU] [<span class="glyphicon glyphicon-remove"/> Unsubscribe][C_UN] | |
-| **notifications** | Automated notifications (GitHub, etc.)             | [<span class="glyphicon glyphicon-search"/> Archive][N_A] | [<span class="glyphicon glyphicon-plus"/> Subscribe][N_SU] [<span class="glyphicon glyphicon-remove"/> Unsubscribe][N_UN] | |
+| **user**          | General user questions, help, and announcements  | [<i class="fa-solid fa-magnifying-glass"></i> Archive][U_A] | [<i class="fa-solid fa-plus"></i> Subscribe][U_SU] [<i class="fa-solid fa-xmark"></i> Unsubscribe][U_UN] | [<i class="fa-solid fa-envelope"></i> Post][U_P] |
```

Comment:
`<i>` is for icons not italicizing afaik. Thats how its listed on the fontawesome icons docs: https://fontawesome.com/icons/plus?f=classic&s=solid

Reply:
Nah, it's defined as "alternate voice or mood", typically presented as italics. There was an attempt with html4 to replace it with `<em>` to semantically mean "emphasis" and leave `<i>` to literally mean italics, but it never caught on because nobody ever styled `<em>` differently than `<i>`, and it was just a redundant tag. I think it was commonly used in html4 to represent an arbitrary inline element, because not all browsers consistently displayed `<span>` elements that were effectively empty. With html5, that's not really an issue anymore, because of the more widespread popularity of modern versions of CSS.

Reply:
I think one reason why people might prefer `<i>` is because it's shorter, and people seem to like minimizing. I don't find it helps that much, and I think it obfuscates things too much. Plus, if I actually want to put some text in there, like `<i class="fa-solid fa-magnifying-glass">something that wraps with the magnifying glass</i>`, then I have to specifically add CSS to *un*-style it.

Reply:
I see. Thanks for explaining that. I can go ahead and change these to `<span>`. I was just going off of the fontawesome docs.

Reply:
Resolved in 4bb6421

Reply:
My main goal in asking about this was mostly to suggest not making unnecessary changes, in order to keep the diff minimal. As long as there's a reason to choose one over the other... whether it's "keep the diff minimal", "follow the bootstrap examples", "keep the code small", "consistency", or some other reason. I just might ask about those reasons, but that's not necessarily an objection.

Reply:
> My main goal in asking about this was mostly to suggest not making unnecessary changes, in order to keep the diff minimal. As long as there's a reason to choose one over the other... whether it's "keep the diff minimal", "follow the bootstrap examples", "keep the code small", "consistency", or some other reason. I just might ask about those reasons, but that's not necessarily an objection.

I didn't have much of an opinion one way or another and you made some good points about using span instead of `<i>` so I think they are fine to stay as `<span>`